XIVLauncher/Windows/MainWindow.xaml.cs

```diff
@@ -251,7 +251,22 @@ private void HandleBootCheck(Action whenFinishAction)
             {
                 App.Settings.PatchPath ??= new DirectoryInfo(Path.Combine(Paths.RoamingPath, "patches"));
 
-                var bootPatches = _launcher.CheckBootVersion(App.Settings.GamePath);
+                Game.Patch.PatchList.PatchListEntry[] bootPatches = null;
+                try
+                {
+                    bootPatches = _launcher.CheckBootVersion(App.Settings.GamePath);
+                }
+                catch (Exception ex)
+                {
+                    Log.Error(ex, "Unable to check boot version.");
+                    MessageBox.Show(Loc.Localize("CheckBootVersionError", "XIVLauncher was not able to check the boot version for the select game installation. This can happen if a maintenance is currently in progress or if your connection to the version check server is not available. Please report this error if you are able to login with the official launcher, but not XIVLauncher."), "XIVLauncher", MessageBoxButton.OK, MessageBoxImage.Error);
+
+                    _ = Task.Run(SetupHeadlines);
```

Comment:
Why the discard?

Reply:
It was just to suppress the warnings in the Task, not needed there though.